auth_totp/README.rst

```diff
@@ -0,0 +1,103 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========
+MFA Support
+===========
+
+This module adds support for MFA using TOTP (time-based, one-time passwords). 
+It allows users to enable/disable MFA and manage authentication apps/devices 
+via the "Change My Preferences" view and an associated wizard. After logging 
+in normally, users with MFA enabled are taken to a second screen where they 
+have to enter a password generated by one of their authentication apps and 
+are presented with the option to remember the current device. This creates a 
+secure, HTTP-only cookie that allows subsequent logins to bypass the MFA step.
+
+Installation
+============
+
+To install this module, simply follow the standard install process, making 
+sure that the PyOTP library is available in your environment. The module will 
+not function properly without PyOTP, which can easily be added via pip: 
+``pip install pyotp``.
+
+Configuration
+=============
+
+By default, the trusted device cookies introduced by this module have a 
```

Comment:
There are only two sentences here. Does each one really need to be a separate paragraph?